integration_test/third_party_apps_data/applications/postgresql/sles/install

```diff
@@ -5,31 +5,19 @@ SUSE_VERSION="${VERSION_ID%%.*}"
 
 if [[ "${SUSE_VERSION}" == 12 ]]; then
   sudo zypper addrepo https://download.postgresql.org/pub/repos/zypp/repo/pgdg-sles-12-pg14.repo
-
   sudo zypper --non-interactive --no-gpg-checks ref
   sudo zypper --no-gpg-checks in -y postgresql14-server=14.1-3.3.1 postgresql14-contrib=14.1-3.3.1 postgresql14-libs=14.1-1PGDG.sles12 postgresql14=14.1-3.3.1
-
-  sudo service postgresql restart
-
-  sudo su postgres -c "psql postgres -c \"ALTER ROLE postgres WITH PASSWORD 'abc123';\""
-  sudo sh -c 'echo "host    all             all             127.0.0.1/32            scram-sha-256" > /var/lib/pgsql/data/pg_hba.conf'
-  sudo sh -c 'echo "host    all             all             ::1/128                 scram-sha-256" >> /var/lib/pgsql/data/pg_hba.conf'
-
-  sudo service postgresql restart
 elif [[ "${SUSE_VERSION}" == 15 ]]; then
-  # NOTE - This should be reverted to a simple addrepo once https://download.postgresql.org/pub/repos/zypp/14/suse/ contains a repository for 15.4
-  curl https://download.postgresql.org/pub/repos/zypp/repo/pgdg-sles-15-pg14.repo -o pg14.repo
-  sed -i 's/\$releasever/15.3/g' pg14.repo
-  sudo zypper addrepo pg14.repo
-
   sudo zypper --non-interactive --no-gpg-checks ref
-  sudo zypper --no-gpg-checks in -y postgresql14-server postgresql14-contrib postgresql14-libs postgresql14
+  sudo zypper --no-gpg-checks in -y postgresql14-server postgresql14-contrib postgresql14
+fi
 
-  sudo systemctl enable postgresql-14
+sudo systemctl enable postgresql
+sudo service postgresql restart
 
-  sudo /usr/pgsql-14/bin/postgresql-14-setup initdb
-  sudo service postgresql-14 restart
+sudo su postgres -c "psql postgres -c \"ALTER ROLE postgres WITH PASSWORD 'abc123';\""
 
-  sudo su postgres -c "psql postgres -c \"ALTER ROLE postgres WITH PASSWORD 'abc123';\""
-fi
+sudo sh -c 'echo "host    all             all             127.0.0.1/32            scram-sha-256" > /var/lib/pgsql/data/pg_hba.conf'
+sudo sh -c 'echo "host    all             all             ::1/128                 scram-sha-256" >> /var/lib/pgsql/data/pg_hba.conf'
 
+sudo service postgresql restart
```

Comment:
Do we need the second `restart` to pick up the content of `pg_hba.conf`? Can we instead create that file before the first `restart`?

Reply:
Turns out we have to start postgres to generate that file so I added the start before the file insertion